File: Swift/BOJ/15000/17219.swift
//
//  17219.swift
//  Algorithm
//
//  Created by 유정주 on 2023/06/13.
//

import Foundation

let nm = readLine()!.split { $0 == " " }.map { Int($0)! }
let n = nm[0], m = nm[1]

var dict: [String: String] = [:]
for _ in 0..<n {
    let input = readLine()!.split { $0 == " " }.map { String($0) }
    dict[input[0]] = input[1]
}

for _ in 0..<m {
    let input = readLine()!
    print(dict[input]!)
}

File: Python/BOJ/해시/17219.py
//
//  17219.py
//  SwiftAlgorithm
//
//  Created by 유정주 on 2023/06/13.
//

import sys; readLine = sys.stdin.readline

n, m = map(int, readLine().split())
dict = {}
for _ in range(n):
    input = list(readLine().rstrip().split())
    dict[input[0]] = input[1]

for _ in range(m):
    input = readLine().rstrip()
    print(dict[input])
